### ISSUE TITLE
Escaping code blocks

### ISSUE THREAD

**Issue by Cologler** [darkriszty/MarkdownTablePrettify-VSCodeExt#16]
This table cannot be format:

``` md
sub cmd|description|input|output
:---|:---|:---|:---
skip|skip number of char in input|`echo abc|str skip 1`|`bc`
take|take number of char in input|`echo abc|str take 1`|`a`
```

some `|` was in code block so it should not detect as table separator.

**Comment by darkriszty** [comment]
Hi @Cologler 

Currently there isn't any implementation for escaping the separators, so I took the liberty to rename the issue. There is a request to implement escaping with backslash (meaning \|) and it is described in issue #11 . I will think about this code block escaping as well.

**Comment by Cologler** [comment]
In vscode, I don't need escaped char:

![picpick-20170816-211144-001](https://user-images.githubusercontent.com/10906962/29364890-8eb24c30-82c7-11e7-8ec7-b8dc285447d2.png)

Vscode parse ignore all `|` in code block.

**Comment by darkriszty** [comment]
Included in v2.0.0
